test/unit/mdc-drawer/mdc-drawer.test.js

```diff
@@ -29,19 +29,38 @@ import td from 'testdouble';
 import {MDCDrawer} from '../../../packages/mdc-drawer/index';
 import {strings, cssClasses} from '../../../packages/mdc-drawer/constants';
 import {MDCListFoundation} from '../../../packages/mdc-list/index';
-import MDCDismissibleDrawerFoundation from '../../../packages/mdc-drawer/dismissible/foundation';
+import {MDCDismissibleDrawerFoundation} from '../../../packages/mdc-drawer/dismissible/foundation';
+import {MDCModalDrawerFoundation} from '../../../packages/mdc-drawer/modal/foundation';
 
+/**
+ * @typedef {{
+ *   variantClass: (string|undefined),
+ *   shadowRoot: (boolean|undefined),
+ *   hasList: (boolean|undefined),
+ * }}
+ */
+let DrawerSetupOptions; // eslint-disable-line no-unused-vars
```

Comment:
Why do we need this? We're not running unit tests through closure compiler, right?

Reply:
Correct, tests don't get run through Closure Compiler. However, IDEs understand JSDoc syntax and provide better code completion, navigation, and "find usages" when these comments are present. Our tests are so poorly typed that most symbols have squiggly lines under them because the editor has no idea what the type of anything is. I added this to make my life easier while debugging these tests.